plutus-core/test-cost-model/TestCostModels.hs

```diff
@@ -48,24 +48,24 @@ prop_greaterThanEqInteger :: Property
 prop_greaterThanEqInteger = testPredict greaterThanEqInteger (getConst . paramGreaterThanEqInteger)
 prop_eqInteger :: Property
 prop_eqInteger = testPredict eqInteger (getConst . paramEqInteger)
--- prop_concatenate :: Property
--- prop_concatenate = testPredict concatenate (getConst . paramConcatenate)
--- prop_takeByteString :: Property
--- prop_takeByteString = testPredict takeByteString (getConst . paramTakeByteString)
--- prop_dropByteString :: Property
--- prop_dropByteString = testPredict dropByteString (getConst . paramDropByteString)
--- prop_sha2 :: Property
--- prop_sha2 = testPredict sHA2 (getConst . paramSha2)
--- prop_sha3 :: Property
--- prop_sha3 = testPredict sHA3 (getConst . paramSha3)
+prop_concatenate :: Property
+prop_concatenate = testPredict concatenate (getConst . paramConcatenate)
+prop_takeByteString :: Property
+prop_takeByteString = testPredict takeByteString (getConst . paramTakeByteString)
+prop_dropByteString :: Property
+prop_dropByteString = testPredict dropByteString (getConst . paramDropByteString)
+prop_sha2 :: Property
+prop_sha2 = testPredictOne sHA2 (getConst . paramSHA2)
+prop_sha3 :: Property
+prop_sha3 = testPredictOne sHA3 (getConst . paramSHA3)
 -- prop_verifySignature :: Property
 -- prop_verifySignature = testPredict verifySignature (getConst . paramVerifySignature)
```

Comment:
Didn't you do `verifySignature` in this PR?

Reply:
Oops. Fixed.